templates/lxc-busybox.in

```diff
@@ -293,21 +300,25 @@ do
     --rootfs)     rootfs=$2; shift 2;;
     --mapped-uid) LXC_MAPPED_UID=$2; shift 2;;
     --mapped-gid) LXC_MAPPED_GID=$2; shift 2;;
+    --bbpath) BUSYBOX_EXE=$2; shift 2;;
```

Comment:
Maybe `--busybox-path` is better than `--bbpath`?

Reply:
I agree. It follows thez naming scheme other options like "mapped-uid".